build/AssemblyRedirectionSourceGenerator.cs

```diff
@@ -35,39 +35,48 @@ public static void Generate(string assembliesFolderPath, string generatedFilePat
         Log.Information("Assembly redirection source generated {0}", generatedFilePath);
     }
 
-    static string GenerateSourceContents(SortedDictionary<string, AssemblyNameDefinition> assemblies)
+    private static string GenerateSourceContents(SortedDictionary<string, AssemblyNameDefinition> assemblies)
     {
-        var sb = new StringBuilder(assemblies.Count * 256);
-        sb.AppendLine("""
-    /*
-     * Copyright The OpenTelemetry Authors
-     * SPDX-License-Identifier: Apache-2.0
-     */
-    """);
-        sb.AppendLine();
-        sb.AppendLine($"// Auto-generated file, do not change it - generated by the {nameof(AssemblyRedirectionSourceGenerator)} type");
-        sb.Append(@"
-#include ""cor_profiler.h""
+        return
+        $$"""
+        /*
+         * Copyright The OpenTelemetry Authors
+         * SPDX-License-Identifier: Apache-2.0
+         */
+
+        // Auto-generated file, do not change it - generated by the {{nameof(AssemblyRedirectionSourceGenerator)}} type
+
+        #include "cor_profiler.h"
+
+        #ifdef _WIN32
+        namespace trace
+        {
+        void CorProfiler::InitNetFxAssemblyRedirectsMap()
+        {
+            assembly_version_redirect_map_.insert({
+                {{GenerateEntries(assemblies)}}
```

Comment:
The indentation difference might be a difference between tabs and spaces. I didn't pull the file locally, but at least in the web view it looks like spaces are used here, and tabs are used by GenerateEntries. The leading tabs for the first line is removed by the call to trim.

Reply:
Yep, converted back to spaces, that fixed the issue. Somehow locally no difference 🤷